package.json

```diff
@@ -4,11 +4,15 @@
   "version": "0.0.0",
   "repository": "git@github.com:backstage/community-plugins.git",
   "scripts": {
-    "create-workspace": "node ./scripts/create/create.js"
+    "create-workspace": "node ./scripts/create/create.js",
+    "detect-workspace": "node ./scripts/ci/changes.js"
```

Comment:
Is this one worth having here? Not really for end users to run. Might as well just call this command directly inside the CI script I think.

May also make sense to name these `create/create-workspace.js` and `ci/list-workspaces-with-changes.js` or so, helps find the right one in the future